### DISCUSSION TITLE
Hi. How can I fix the "Hydratition node mismatch"

### DISCUSSION THREAD

**Discussion by Muhammad19944** [nuxt/nuxt#19048] [Questions]
This warning shown after updating nuxt project version 3.2.0

`{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "nuxt": "^3.2.0"
  }
}`

![image](https://user-images.githubusercontent.com/31342676/218953469-8a17656c-b4f9-4c4d-84f0-c278b8c00fe9.png)

![image](https://user-images.githubusercontent.com/31342676/218953532-d5251a14-2477-4767-925c-494ac1ddd750.png)

**Comment by TheAlexLichter** [comment]
Might be https://github.com/nuxt/nuxt/issues/18635

**Reply by Muhammad19944** [reply]
it did not help (

**Reply by Muhammad19944** [reply]
Because of what it can be? The project is still new
